client/packages/common/src/ui/components/inputs/TextInput/NumericTextInput.tsx

```diff
@@ -67,14 +67,17 @@ export const NumericTextInput: FC<NumericTextInputProps> = React.forwardRef(
     const isFirstRender = useRef(true);
 
     const isInputIncomplete = useCallback(
-      (value: string) =>
-        new RegExp(
+      (value: string) => {
+        if (value === '-') return true;
+
```

Comment:
Not sure how long this has been broken. I guess we don't use negative values much in oms.